Tensile/Source/lib/include/Tensile/ContractionProblemPredicates.hpp

```diff
@@ -550,33 +550,6 @@ namespace Tensile
                 }
             };
 
-            struct CDStridesEqual : public Predicate_CRTP<CDStridesEqual, ContractionProblem>
```

Comment:
shall we just keep CDStridesEqual() but create new CDStridesEqualInMultiFreeA()?

Reply:
rename back to CDStridesEqual